src/Component/src/Exception/DeleteHandlingException.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Exception;
+
+class DeleteHandlingException extends \Exception
+{
+    protected string $flash;
+
+    protected int $apiResponseCode;
+
+    public function __construct(
+        string $message = 'Ups, something went wrong during deleting a resource, please try again.',
```

Comment:
Was it meant to be like so?
```suggestion
        string $message = 'Oops, something went wrong during deleting a resource, please try again.',
```
How about,
```suggestion
        string $message = 'Something went wrong during deleting a resource, please try again.',
```